docs/UserGuide.md

```diff
@@ -177,19 +177,21 @@ the warehouse at index 1 of the list of warehouses.
 * `delete supplier 12 pd/Aspirin` Removes product with the name _Aspirin_
 from the supplier at index 12 of the list of suppliers.
 
-### Finding medical product associated with warehouses / suppliers: `find`
+### Finding suppliers/warehouses: `find`
 
-Finds all suppliers or warehouses managed by the manager that sells the relevant medical products.
+Finds all suppliers or warehouses whose name, remark and/or products sold/stored matches the provided argument keywords.
 
-Format: `find TYPE KEYWORD [KEYWORD]…​`
+Format: `find ct/TYPE [n/NAME…​] [pd/PRODUCT_NAME…​] [r/REMARK…​]`
 
-* KEYWORD specified is case-insensitive and must contain at least one keyword.
-* The TYPE specified should be one of these values: warehouse / supplier.
+* Keywords specified are case-insensitive.
+* The TYPE specified should be one of these values: s/w.
+* Any combination of the name, product and remark prefix can be provided but at least one of the prefix for name,
+product and remark must be specified.
 
 Examples:
 
-* `find warehouse PANADOL SUSP` Displays all the warehouses managed by the manager that has a product named PANADOL SUSP.
-* `find supplier masks` Displays all the suppliers that have stock for the input product.
+* `find ct/s pd/masks` Displays all the suppliers that sell masks.
+* `find ct/w pd/PANADOL SUSP r/biggest` Displays the biggest warehouse that stores PANADOL SUSP.
```

Comment:
Should it display the biggest warehouse with any PANADOL product or SUSP product instead?